src/diracx/db/jobs/db.py

```diff
@@ -260,6 +271,37 @@ async def get_job_status(self, job_id: int) -> LimitedJobStatusReturn:
             **dict((await self.conn.execute(stmt)).one()._mapping)
         )
 
+    async def set_job_command(self, job_id: int, command: str, arguments: str = ""):
+        """Store a command to be passed to the job together with the next heart beat"""
+        stmt = insert(JobCommands).values(
+            JobID=job_id,
+            Command=command,
+            Arguments=arguments,
+            ReceptionTime=datetime.now(tz=timezone.utc),
+        )
+        await self.conn.execute(stmt)
+
+    async def get_vo(self, job_id: int) -> str:
+        """
+        Get the VO of the owner of the job
+        """
+        # TODO: Consider having a VO column in the Jobs table
+        from DIRAC.Core.Utilities.ClassAd.ClassAdLight import ClassAd
+
+        # TODO: this is going to be problematic
```

Comment:
Can you clarify exactly what you mean here? Just the performance or something else?